routing_model/index.html

```diff
@@ -433,37 +439,181 @@ <h2>Process for Adult/Elder promotion and demotion</h2>
 
 
       WaitFor(("Wait for 5:"))
-       LoopStart --> WaitFor
+      LoopStart --> WaitFor
 
       WaitFor -- Event --> Event
       Event((Event))
       Event-- CheckElder<br/>Timeout--> VoteCheckElderTimeout
       VoteCheckElderTimeout["vote_for(<br/>ParsecCheckElderTimeout)"]
       VoteCheckElderTimeout--> LoopEnd
 
+      RPC((RPC))
+      WaitFor -- RPC --> RPC
+      RPC --Merge--> VoteParsecNeighbourMerge
+      VoteParsecNeighbourMerge["vote_for(<br/>ParsecNeighbourMerge)"]
+      VoteParsecNeighbourMerge --> LoopEnd
+
       Consensus((Consensus))
       WaitFor-- Parsec<br/>consensus --> Consensus
-      Consensus--"ParsecCheckElderTimeout"-->CheckElderChange
-      CheckElderChange(("check_elder_change()"))
-      CheckElderChange -- "Has elder changes: elder first ordered by:<br/>State=Online then age then name." --> MarkAndVoteSwapNewElder
-      MarkAndVoteSwapNewElder["vote_for(Add) for new elders<br/>vote_for(Remove) for now adults nodes<br/>vote_for(NewSectionInfo)<br/><br/>WAITED_VOTES.insert(all votes)"]
-      MarkAndVoteSwapNewElder -->LoopEnd
 
+      Consensus -- "ParsecNeighbourMerge" --> SetNeighbourMerge
+      SetNeighbourMerge["store_merge_infos(info)"]
```

Comment:
This "info" hasn't been defined anywhere before and isn't a local variable (INFO). Should we say: "where info is contained in the ParsecNeighbourMerge event", or such?

Reply:
Updated with: `store_merge_infos(ParsecNeighbourMerge info)`
There was also a function documentation: ` store_merge_infos: Store the neighbour merge info, may not be sibling in case of multi merge`